lib/checks/landmarks/landmark-is-unique.json

```diff
@@ -0,0 +1,16 @@
+{
+	"id": "landmark-is-unique",
+	"evaluate": "landmark-is-unique.js",
+	"after": "landmark-is-unique-after.js",
+	"tags": ["best-practice"],
+	"options": {
+		"selector": "[role=banner], [role=complementary], [role=contentinfo], [role=main], [role=navigation], [role=region], [role=search], [role=form], form, footer, header, aside, main, nav, section"
+	},
+	"metadata": {
+		"impact": "moderate",
+		"messages": {
+			"pass": "Landmarks must have a unique role or role/label combination(aria-label OR aria-labelledby).",
+			"fail": "The landmarks do not have a unique role or role/label combination (use aria-label OR aria-labelledby to make landmarks distinguishable)."
```

Comment:
As before, I'm not sure why this isn't considering title. That certainly seems to work as a way to set the accessible name of a landmark.

Reply:
Updated.